src/css/consultancy-page.css

```diff
@@ -0,0 +1,121 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+   }
+  /* To reset some default browser styles */
+   body, p , ul, li {
+    margin: 0 ;
+    padding: 0 ;
+   }
```

Comment:
this section is repetitive since *{} applies to all elements, or explain why its needed.